medical_lab/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2004-2009 Tiny SPRL
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
```

Comment:
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py

Ordered improperly + mix of single and double quotes. Should all be double.